pkg/backend/display/testdata/not-truncated/stack-output-resource-diff.json.interactive-100x80-cooked.stdout.txt

```diff
@@ -0,0 +1,56 @@
+ <{%bold%}><{%reset%}>  <{%reset%}>  pulumi:pulumi:Stack    pulumi-stack-output-diff-dev  running<{%bold%}><{%reset%}><{%reset%}>     <{%reset%}>Configuration:<{%reset%}>[K
+ <{%reset%}>  <{%reset%}>  ├─ some:demo:Taxi      taxi                          <{%reset%}><{%reset%}>            [K
+ <{%reset%}>  <{%reset%}>  └─ some:demo:Vehicles  vehicles                      <{%reset%}><{%reset%}>            [K
+
[1A[1A[1A[1A     <{%underline%}><{%fg 12%}>Type<{%reset%}>                   <{%underline%}><{%fg 12%}>Name<{%reset%}>                          <{%underline%}><{%fg 12%}>Status<{%reset%}>      <{%underline%}><{%fg 12%}>Info<{%reset%}>[K
+ <{%bold%}><{%reset%}>  <{%reset%}>  pulumi:pulumi:Stack    pulumi-stack-output-diff-dev  running<{%bold%}><{%reset%}><{%reset%}>     <{%reset%}>Configuration:<{%reset%}>[K
+ <{%reset%}>  <{%reset%}>  ├─ some:demo:Taxi      taxi                          <{%reset%}><{%reset%}>            [K
+ <{%reset%}>  <{%reset%}>  └─ some:demo:Vehicles  vehicles                      <{%reset%}><{%reset%}>            [K
+
[1A[1A[1A[1A     <{%underline%}><{%fg 12%}>Type<{%reset%}>                   <{%underline%}><{%fg 12%}>Name<{%reset%}>                          <{%underline%}><{%fg 12%}>Status<{%reset%}>     <{%underline%}><{%fg 12%}>Info<{%reset%}>[K
+ <{%reset%}>  <{%reset%}>  pulumi:pulumi:Stack    pulumi-stack-output-diff-dev  <{%reset%}><{%reset%}>           1 <{%fg 5%}>message<{%reset%}>[K
+ <{%reset%}>  <{%reset%}>  ├─ some:demo:Taxi      taxi                          <{%reset%}><{%reset%}>           [K
+ <{%reset%}>  <{%reset%}>  └─ some:demo:Vehicles  vehicles                      <{%reset%}><{%reset%}>           [K
+
[1A[1A[1A[1A     <{%underline%}><{%fg 12%}>Type<{%reset%}>                   <{%underline%}><{%fg 12%}>Name<{%reset%}>                          <{%underline%}><{%fg 12%}>Status<{%reset%}>     <{%underline%}><{%fg 12%}>Info<{%reset%}>[K
+ <{%reset%}>  <{%reset%}>  pulumi:pulumi:Stack    pulumi-stack-output-diff-dev  <{%reset%}><{%reset%}>           1 <{%fg 5%}>message<{%reset%}>[K
+ <{%reset%}>  <{%reset%}>  ├─ some:demo:Taxi      taxi                          <{%reset%}><{%reset%}>           [K
+ <{%reset%}>  <{%reset%}>  └─ some:demo:Vehicles  vehicles                      <{%reset%}><{%reset%}>           [K
+[K
+
+<{%fg 13%}><{%bold%}>Diagnostics:<{%reset%}>
+  <{%fg 12%}>pulumi:pulumi:Stack (pulumi-stack-output-diff-dev):<{%reset%}>
+    <{%reset%}>Configuration:<{%reset%}>
+
+<{%fg 13%}><{%bold%}>Outputs:<{%reset%}>
+<{%fg 3%}>  ~ vehicles: <{%reset%}><{%fg 3%}>{
+<{%reset%}><{%fg 2%}>      + @isPulumiResource: <{%reset%}><{%fg 2%}>true<{%reset%}><{%fg 2%}>
+<{%reset%}><{%fg 3%}>      ~ taxi             : <{%reset%}><{%fg 3%}>{
+<{%reset%}><{%fg 2%}>          + @isPulumiResource: <{%reset%}><{%fg 2%}>true<{%reset%}><{%fg 2%}>
```

Comment:
the `progress` printer actually shows the `@isPulumiResource` fields, and I think that's fine. The fix is demonstrated in the files produced by `diff_test.go`